examples/parsed_view_func.py

```diff
@@ -13,7 +13,7 @@
 app = Flask(__name__)
 app.config['SWAGGER'] = {
     'title': 'Flasgger Parsed Method/Function View Example',
-    'doc_dir': './examples/docs/'
```

Comment:
Ideally, we want to maintain backward compatibility, even in our examples, but I would agree with your change to promote this new feature.